src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -121,4 +123,14 @@ public static Tag parseTag(String tag) throws ParseException {
         }
         return tagSet;
     }
+
+    /**
+     * Parses {@code String date} into a {@code Date date}.
+     * @throws java.text.ParseException if the given {@code String date} is invalid.
+     */
```

Comment:
`ParserUtil` class imports `logic.parser.exceptions.ParseException` so if I were to throw just `ParseException` it would refer to the exception from `logic.parser.exceptions` and not `java.text`